src/test/features/cucumberSteps.feature

```diff
@@ -0,0 +1,70 @@
+Feature: JSON Placeholder
+
+  @automated
+  Scenario Outline:  Send GET Request to get all posts (/posts).
+    
+    When the user send get request to '/posts'
+    Then the API status code should be 200 - OK
+    And the response is in json format
+    And the response posts are sorted ascending by id
+
+  @automated
+  Scenario Outline: Send GET request to get post with id=99 (/posts/99).
+    
+    When the user send get request to '/posts' with '<id>' id
+    Then the API status code should be 200 - OK
+    And the get response 'id' is match '<id>'
+    And the get response 'userId' is match '<userId>'
+    And the 'title' is not empty
+    And the 'body' is not empty
+
+    Examples:
+      | id  | userId |
+      | 99  |   10   |
```

Comment:
if there is only one example then examples table can be removed and Scenario used instead of Scenario Outline